src/components/ListView/index.jsx

```diff
@@ -1,22 +1,60 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 import { useCollection } from 'react-firebase-hooks/firestore';
-import { getProducts } from 'components/Utils/firestore';
+import {
+  getProducts,
+  convertCollectionToArray,
+} from 'components/Utils/firestore';
+import { isThereCoincidence } from 'components/Utils/helpers';
 import ProductList from 'components/ProductList';
 import EmptyList from 'components/EmptyList';
 
 function ListView() {
+  const [productsFiltered, setProductsFiltered] = useState([]);
+  const [nameFilter, setNameFilter] = useState('');
   const query = getProducts();
   const [values, loading, error] = useCollection(query);
+  const handleInput = (e) => setNameFilter(e.target.value);
+
+  useEffect(() => {
+    const collectionProducts = values ? values.docs : [];
+    const arrayProducts = convertCollectionToArray(collectionProducts);
+    setProductsFiltered(arrayProducts);
+    if (!loading && nameFilter.length > 0) {
+      setProductsFiltered(
+        arrayProducts.filter((product) =>
+          isThereCoincidence(product.item, nameFilter),
```

Comment:
Similar to the other comment, I would rename this function. Maybe to isSubstring?

Reply:
Thanks, we changed `isThereCoincidence `to `isSubstring`.